clippy.toml

```diff
@@ -0,0 +1 @@
+type-complexity-threshold = 500
```

Comment:
This seems a bit high. What's wrong with the default?

Reply:
It doesn't work with Specs. We make heavy usage of tuples with 10 or more fields, which clippy doesn't like. 500 does seem high, though.

Reply:
You can `#[allow(clippy::type_complexity)]` for those types.

Reply:
Yes, that's right. I assume this line was added because adding such an attribute to every single `System` implementation would be quite annoying.